src/store/index.js

```diff
@@ -130,6 +130,7 @@ const initializeAppHolo = async (commit, dispatch, state) => {
         logo_url: "img/ECLogoWhiteMiddle.png",
         app_name: "Elemental Chat",
         info_link: "https://holo.host/faq-tag/elemental-chat"
+        // publisher_name: "Holo"
```

Comment:
do we need this comment?

Reply:
I guess I should add that to the Web SDK README instead of leaving it there. Fair enough